todo-app/src/component/todos.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import TodoItems from './todoItems';
+
+const Todos = ({ todos }) => {
```

Comment:
Nice use of a functional component!

Just a little trick, you can still do your exporting inline and have an unnamed default export:

    export default ({ todos }) => {